src/frontend/sharedComponents/ActionRow.tsx

```diff
@@ -59,5 +84,14 @@ export const ActionsRow = ({fieldRefs}: ActionButtonsProps) => {
     });
   }
 
-  return <ActionTab items={bottomSheetItems} />;
+  return (
+    <>
+      <ActionTab items={bottomSheetItems} />
+      <PermissionAudio
+        closeSheet={closeAudioPermissionSheet}
+        isOpen={isAudioPermissionSheetOpen}
+        sheetRef={audioPermissionSheetRef}
+      />
```

Comment:
I think it would be better to have the `<BottomSheetModal / >` component live on this page, so you don't need to pass the props down (that's how it is designed to be used)

And then I think you could rename the `PermissionAudio` to be more descriptive such as `PermissionAudioBottomSheetContent`

So it would look like
```ts
< BottomSheetModal closeSheet={closeAudioPermissionSheet}
    isOpen={isAudioPermissionSheetOpen}
    sheetRef={audioPermissionSheetRef} >
        < PermissionAudioBottomSheetContent />
</ BottomSheetModal>
```

Reply:
Ok.